src/ziggurat/metrics.clj

```diff
@@ -76,22 +81,32 @@
    (inc-or-dec-count sign {:metric-namespace metric-namespace :metric metric :n n :additional-tags additional-tags}))
   ([sign {:keys [metric-namespace metric n additional-tags]}]
    (let [metric-ns (get-metric-namespaces metric-namespace)
-         meter     ^Meter (mk-meter metric-ns metric (get-map additional-tags))]
+         meter     ^Meter (mk-meter metric-ns metric (remove-topic-tag-for-old-namespace (get-map additional-tags) metric-namespace))]
      (.mark meter (sign (get-int n))))))
 
 (def increment-count (partial inc-or-dec-count +))
 
 (def decrement-count (partial inc-or-dec-count -))
 
+(defn multi-ns-increment-count [nss metric additional-tags]
```

Comment:
We should have a unit test for multi-ns-increment-count

Reply:
added

Reply:
done